docs/source/conf.py

```diff
@@ -6,14 +6,19 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
+
+# Get the commit hash from git
+import subprocess
 import sys
 from pathlib import Path
 
+commit_id = subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode("utf-8")
+print(f"Git Commit ID: {commit_id}")
```

Comment:
where does this print go, should we be logging it?

Reply:
The print goes in the github action. We could remove it, I"m fine with either way